src/components/edit-on-github/edit-on-github.js

```diff
@@ -0,0 +1,33 @@
+import styles from "./edit-on-github.module.css";
+
+const REPO_PREFIX = "https://github.com/ProjectEvergreen/www.greenwoodjs.dev/blob/main/src/pages/";
```

Comment:
I wasn't sure if it was meant to go directly to the edit page, or to the read-only view. It's an easy change if required. ".../blob/..." => ".../edit/..."